src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc

```diff
@@ -389,13 +392,10 @@ void PickFirst::PickFirstSubchannelData::ProcessConnectivityChangeLocked(
     case GRPC_CHANNEL_READY:
       // Already handled this case above, so this should not happen.
       GPR_UNREACHABLE_CODE(break);
-    case GRPC_CHANNEL_TRANSIENT_FAILURE:
-    case GRPC_CHANNEL_IDLE: {
-      PickFirstSubchannelData* sd = this;
-      size_t next_index =
-          (sd->Index() + 1) % subchannel_list()->num_subchannels();
+    case GRPC_CHANNEL_TRANSIENT_FAILURE: {
+      size_t next_index = (Index() + 1) % subchannel_list()->num_subchannels();
       subchannel_list()->set_attempting_index(next_index);
-      sd = subchannel_list()->subchannel(next_index);
+      PickFirstSubchannelData* sd = subchannel_list()->subchannel(next_index);
```

Comment:
No, just making the code flow a little more readably.  There was no reason to initially assign `sd` to `this` and then say `sd->Index()` instead of just saying `Index()` above.  This isn't a functional change.